utils/paste.tsx

```diff
@@ -1,5 +1,8 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+import TurndownService from 'turndown';
+import {tables} from 'turndown-plugin-gfm';
```

Comment:
just use this, since `turndown-plugin-gfm` does not support esm:

```suggestion
const tables = require('turndown-plugin-gfm').tables;
```

Reply:
It would be great if you could add a small comment on top of this line explaining why `require` is used instead of `import`

Reply:
Not sure what the differences are between these, left this for now

Reply:
@hmhealey I just wanted to keep the amount of new file creations at a minimum... 

@EricssonLiu it is just how the module is resolved ... esm is the easiest, since you can just use `import`. `require` is the 'old' way of doing it! :D